docs/community/contributing.rst

```diff
@@ -18,7 +18,14 @@ Get connected
 -------------
 
 First off, get yourself connected with the documentation community online. There
-are a few places where we have conversations and discussion.
+are a few places where we have conversations and discussion:
+
+- `The Python Discourse category <https://discuss.python.org/c/core-dev/documentation/26>`_
+  is the main place for discussions.
```

Comment:
```suggestion
  is the main hub for discussions.
```

More appropriate and compelling diction and avoid repetition with the preceding line